Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.6-alpine as build
+
+RUN apk add --no-cache g++ musl-dev make
+
+WORKDIR /app
+
+ADD Gemfile* /app/
+RUN gem install bundler \
+    && bundle config build.nokogiri --use-system-libraries \
+    && bundle install --frozen --jobs 4 --retry 5
+
+COPY ./docker-entrypoint.sh /docker-entrypoint.sh
+
+COPY . .
+
+ENTRYPOINT ["/docker-entrypoint.sh"]
```

Comment:
(sorry for the perhaps stupid question, but) is it really necessary to do this? I mean, the [Gemfile](https://github.com/openstreetmap/stateofthemap-2019/pull/46/files#diff-8b7db4d5cc4b8f6dc8feb7030baa2478) has exactly one dependency and wouldn't change very often. And in case it does, a dev container could just be restarted to have its dependencies [refreshed](https://github.com/openstreetmap/stateofthemap-2019/pull/46/files#diff-3254677a7917c6c01f55212f86c57fbfR8-R10), couldn't it?

Reply:
Yes, it is a bit overkill at the moment. I could see `Gemfile` being updated to lock down version (eg: 3.x) or plugins like `jekyll-feed` being added.